src/wiki/plugins/images/models.py

```diff
@@ -108,7 +108,12 @@ def on_image_revision_delete(instance, *args, **kwargs):
     if not instance.image:
         return
     # Remove image file
-    path = instance.image.path.split("/")[:-1]
+    try:
+        path = instance.image.path.split("/")[:-1]
+    except NotImplemented as e:
```

Comment:
In this line, you shouldn't assign `e` since it isn't used, it's possible to merely do `except NotImplemented`.

After changing it, try using:

```
git add src/wiki/plugins/images/models.py
git commit --amend
git push -f
```

This will overwrite the previous commit and force-push to the remote branch, so you don't have to create a new PR.

Nice with the comment below :+1:

Reply:
yeah, good catch, at first I thought about logging it but there is little use of it.